cli/endpoints/online/managed/vnet/mlflow/model/conda.yaml

```diff
@@ -2,9 +2,9 @@ name: mlflow-env
 channels:
 - conda-forge
 dependencies:
-- python=3.7.12
+- python=3.9
 - pip
 - pip:
   - mlflow
-  - cloudpickle==2.0.0
-  - scikit-learn==1.0.2
+  - cloudpickle==2.2.1
+  - scikit-learn==1.2.2
```

Comment:
should we update the model file as we changed the scikit-learn version?

Reply:
Hi @snowei, could you share where and what we should update exactly? Sorry, I'm unfamiliar. :(